notifications.bs

```diff
@@ -294,13 +294,15 @@ removed from the <a>list of notifications</a>.
 <h3 id="permissions-integration">Permissions integration</h3>
 
 <p>The Notifications API is a <a>powerful feature</a> which is identified by the
-<a for="powerful feature">name</a> "{{PermissionName/notifications}}". [[!Permissions]]
+<a for="powerful feature">name</a>
+"<dfn export permission id="notification-permission-name"><code>notifications</code></dfn>".
```

Comment:
There's no documentation for `permission` in https://tabatkins.github.io/bikeshed/. Is that still being worked on?

Reply:
Hmm, yeah. Right now https://github.com/tabatkins/bikeshed/pull/2232#issuecomment-1050411896 is the documentation, but I can send in a PR to get it in the docs.

Reply:
https://github.com/tabatkins/bikeshed/pull/2360